contract_agent_interface/partner.py

```diff
@@ -90,7 +90,7 @@ def _get_analytic_lines(self, cr, uid, ids, field_name,
             'Payments'),
         'all_bank_ids': fields.one2many(
             'res.partner.bank', 'partner_id', 'Banks',
-            domain=['|', ('active', '=', False), ('active', '=', True)],
```

Comment:
Thanks for your explanation. I'll cancel this commit then.

However, after installing the modules from 7.0_contract_agent_interface branch, we get an error message indicating the "active" field doesn't exist for res.partner.bank, hence my merge request. I'll investigate and come back to you if I find more information.